lib/auth/clt.go

```diff
@@ -1170,6 +1170,11 @@ func (c *Client) SearchSessionEvents(fromUTC, toUTC time.Time, limit int, order
 	return events, lastKey, nil
 }
 
+// GetEntitledNode returns a node by name only if user is entitled to it with login name.
+func (c *Client) GetEntitledNode(ctx context.Context, namespace string, name string, login string) (types.Server, error) {
+	return nil, nil
```

Comment:
If we are not implementing throw not implemented error

Reply:
Implemented.